install_config/cluster_metrics.adoc

```diff
@@ -179,8 +176,7 @@ $ oc secrets new metrics-deployer hawkular-metrics.pem=/home/openshift/metrics/h
 hawkular-metrics-ca.cert=/home/openshift/metrics/hm-ca.cert
 ----
 
-You can also use the default auto-generated certificates with the following
-command:
+The Metrics Deployer can accept multiple certificates using secrets. If a certificate is not passed as a secret, the deployer will generate a self-signed certificate to be used instead. For the deployer to generate certificates for you, a secret is still required before it can be deployed. In this case, create a 'dummy' secret that does not specify a certificate value:
```

Comment:
Not sure about the single-quotes around "dummy".  I think either double-quotes or unquoted is better.  I often see it unquoted in technical docs.  It's not mentioned in [this writing style guide](https://obriend.fedorapeople.org/WritingStyleGuide/), which leads me to believe it's sufficiently mainstream.